packages/flutter_test/lib/src/accessibility.dart

```diff
@@ -587,39 +617,43 @@ List<int> _colorsWithinRect(ByteData data, Rect paintBounds, int width, int heig
     final int b = data.getUint8(offset + 2);
     final int a = data.getUint8(offset + 3);
     final int color = (((a & 0xff) << 24) |
-    ((r & 0xff) << 16) |
-    ((g & 0xff) << 8)  |
-    ((b & 0xff) << 0)) & 0xFFFFFFFF;
+            ((r & 0xff) << 16) |
+            ((g & 0xff) << 8) |
+            ((b & 0xff) << 0)) &
+        0xFFFFFFFF;
     return color;
   }
 
-  for (int x = leftX; x < rightX; x ++) {
-    for (int y = topY; y < bottomY; y ++) {
-      buffer.add(_getPixel(data, x, y));
+  for (int x = leftX; x < rightX; x++) {
+    for (int y = topY; y < bottomY; y++) {
+      buffer.update(_getPixel(data, x, y), (int c) => c + 1, ifAbsent: () => 1);
```

Comment:
would help to describe what the rightmost two are doing

Reply:
Which specific part you think more clarification? 
Accumulation or mapping later down the line?

Reply:
I think I get it, default to 1, and count increases by 1?

Reply:
yes, map accumulation. Simplified logic from getting element, checking for null, adding back.